plugins/woocommerce/src/Blocks/Domain/Bootstrap.php

```diff
@@ -141,6 +142,7 @@ function() {
 			$this->container->get( Installer::class )->init();
 			$this->container->get( GoogleAnalytics::class )->init();
 			$this->container->get( CheckoutFields::class )->init();
+			$this->container->get( CheckoutFieldsAdmin::class )->init();
```

Comment:
Would it be better to only load this if `is_admin()` is true? Right now it looks like it is loaded on the front end too

Reply:
I'll revise this in a follow up because I'll want to do the same to the other classes. It won't make much difference performance wise because all init does is hook in.